src/commands/create.js

```diff
@@ -56,7 +56,7 @@ export default asyncCommand({
 		git: {
 			description: 'Initialize version control using git',
 			type: 'boolean',
-			default: true
+			default: false
```

Comment:
I'm not really sure if setting `--git` to false by default or not. 

Leaving `--git` to `true` by default and not making an initial commit makes sense to me.

@developit any thoughts?